src/SwipeableListItem.js

```diff
@@ -226,7 +226,10 @@ class SwipeableListItem extends PureComponent {
 
       switch (octant) {
         case 0:
-          if (horizontalDistance > this.dragHorizontalDirectionThreshold) {
+          if (
+            this.contentRight &&
```

Comment:
Please do not depend on implicit coercion, consider explicit `null` comparison or using a `Boolean` constructor.

```suggestion
            Boolean(this.contentRight) &&
```